ghlib.py

```diff
@@ -45,26 +45,34 @@ def list_hooks_helper(self, entity):
             etype = 'repos'
         else:
             etype = 'orgs'
-        for page in itertools.count(start=1):
-            resp = requests.get(
-                '{api_url}/{etype}/{ename}/hooks?per_page=100&page={page}'.format(
-                    api_url=self.url,
-                    etype=etype,
-                    ename=entity,
-                    page=page
-                ),
-                headers=util.json_accept_header(),
-                auth=self.auth(),
-                timeout=util.REQUEST_TIMEOUT
-            )
-            resp.raise_for_status()
 
-            if not resp.json():
-                break
+        resp = requests.get(
+            '{api_url}/{etype}/{ename}/hooks?per_page=100'.format(
```

Comment:
Looks good. We can add per_page constant or put it in a config just in case we have to change it later

Reply:
Makes sense. I introduced a constant.